resources/monitoring/profile-evaluation.yaml

```diff
@@ -3,13 +3,61 @@ prometheus:
     resources:
       limits:
         cpu: 600m
-        memory: 2000Mi
+        memory: 4Gi
       requests:
         cpu: 75m
         memory: 1000Mi
     retention: 6h
     retentionSize: 1GB
 
+prometheusOperator:
+  namespaces:
+    releaseNamespace: "true"
```

Comment:
I think this is specific to runtimes (SKRs) only where we don't want other namespaces to be watched by prometheus-operator.